lab-integration/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-integration",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/main.js",
+    "dev": "nodemon src/main.ts",
+    "lint": "eslint .  --ext .ts",
+    "lint-fix": "eslint --fix .  --ext .ts"
+  },
+  "devDependencies": {
+    "@stylistic/eslint-plugin-ts": "^1.7.0",
+    "@types/node": "^20.12.7",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "nodemon": "^3.1.0",
+    "prettier": "^3.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^4.3.5"
+  },
+  "dependencies": {
+    "dotenv": "^16.4.5",
+    "puppeteer": "^22.6.4"
```

Comment:
Let's remove it since it's out of the scope and everything else related to this.